lib/components/dashboard/degree-info/CourseBar.tsx

```diff
@@ -62,16 +57,12 @@ const CourseBar: FC<{
   const openCartPopup = async () => {
     dispatch(updateCartInvokedBySemester(false));
     // Filter for the correst distributions from redux store
-    let distr: UserDistribution = distribution;
-    // get fineReqs
-    if (distribution._id) {
-      distr = await getDistribution(distribution._id, token);
-    }
-    if (distr) {
+    let distrs = distributions.filter((req) => req[0] === distribution.name)[0];
+    if (distrs) {
```

Comment:
I think you should just make the total credits bar unclickable by adding a tailwind property to it; that would be in DistributionBarsJSX.tsx 

<img width="1552" alt="Screenshot 2023-03-02 at 2 50 07 PM" src="https://user-images.githubusercontent.com/90944895/222536864-6646158e-8b4f-4438-abf7-2a2b6dc59135.png">

Reply:
that being said, you probably need not touch this file.